ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/vlan-interfaces.j2

```diff
@@ -1,155 +1,169 @@
+{%         elif vlan_interfaces[vlan_interface].shutdown is arista.avd.defined(false) %}
    no shutdown
 {%         endif %}
 {%         if vlan_interfaces[vlan_interface].mtu is defined and vlan_interfaces[vlan_interface].mtu != 1500 %}
    mtu {{ vlan_interfaces[vlan_interface].mtu }}
 {%         endif %}
-{%         if vlan_interfaces[vlan_interface].no_autostate is defined and vlan_interfaces[vlan_interface].no_autostate == true %}
+{%         if vlan_interfaces[vlan_interface].no_autostate is arista.avd.defined(true) %}
    no autostate
 {%         endif %}
-{%         if vlan_interfaces[vlan_interface].vrf is defined and vlan_interfaces[vlan_interface].vrf is not none %}
+{%         if vlan_interfaces[vlan_interface].vrf is arista.avd.defined %}
    vrf {{ vlan_interfaces[vlan_interface].vrf }}
 {%         endif %}
-{%         if vlan_interfaces[vlan_interface].arp_aging_timeout is defined and vlan_interfaces[vlan_interface].arp_aging_timeout is not none %}
+{%         if vlan_interfaces[vlan_interface].arp_aging_timeout is arista.avd.defined %}
    arp aging timeout {{ vlan_interfaces[vlan_interface].arp_aging_timeout }}
 {%         endif %}
-{%             if vlan_interfaces[vlan_interface].ip_proxy_arp is defined and vlan_interfaces[vlan_interface].ip_proxy_arp == true %}
+{%             if vlan_interfaces[vlan_interface].ip_proxy_arp is arista.avd.defined(true) %}
    ip proxy-arp
 {%             endif %}
-{%         if vlan_interfaces[vlan_interface].ip_address is defined %}
+{%         if vlan_interfaces[vlan_interface].ip_address is arista.avd.defined %}
    ip address {{ vlan_interfaces[vlan_interface].ip_address }}
-{%             if vlan_interfaces[vlan_interface].ip_address_secondaries is defined and vlan_interfaces[vlan_interface].ip_address_secondaries is not none %}
+{%             if vlan_interfaces[vlan_interface].ip_address_secondaries is arista.avd.defined %}
 {%                 for ip_address_secondary in vlan_interfaces[vlan_interface].ip_address_secondaries %}
    ip address {{ ip_address_secondary }} secondary
 {%                 endfor %}
 {%             endif %}
 {%         endif %}
-{%         if vlan_interfaces[vlan_interface].ip_virtual_router_address is defined %}
+{%         if vlan_interfaces[vlan_interface].ip_virtual_router_address is arista.avd.defined %}
    ip virtual-router address {{ vlan_interfaces[vlan_interface].ip_virtual_router_address }}
 {%         endif %}
-{%         if vlan_interfaces[vlan_interface].ip_address_virtual is defined %}
+{%         if vlan_interfaces[vlan_interface].ip_address_virtual is arista.avd.defined %}
    ip address virtual {{ vlan_interfaces[vlan_interface].ip_address_virtual }}
 {%         endif %}
-{%         if vlan_interfaces[vlan_interface].ip_helpers is defined and vlan_interfaces[vlan_interface].ip_helpers is not none %}
+{%         if vlan_interfaces[vlan_interface].ip_helpers is arista.avd.defined %}
```

Comment:
This check is included in natural_sort, so we no longer need to have this if outside the for-loops, as long as we use natural_sort.